tools/tscdocgen/main.go

```diff
@@ -267,6 +267,17 @@ func (e *emitter) augmentNode(node *typeDocNode, parent *typeDocNode) {
 	node.Label = createLabel(node, parent)
 	node.CodeDetails = createCodeDetails(node)
 	node.RepoURL = getRepoURL(e.repoURL, node, parent)
+	// ShortText can contain an example snippet that contain placeholders that look like
+	// HTML tags, but aren't. For example, <namespace>/name where <namespace> is the placeholder
+	// for an actual namespace value.
+	if node.Comment.ShortText != "" &&
```

Comment:
The empty check isn't necessary nor are the `Contains` checks, right?

Reply:
Yeah you are right. I don't need them. Just need to always try and replace the `<` and `>`.